lib/model/book.dart

```diff
@@ -1,7 +1,8 @@
 
 import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter/cupertino.dart';
 
-class Book {
+class Book extends ChangeNotifier {
```

Comment:
I think you should document here what the `ChangeNotifier` brings to the Book model